Resources/Locale/en-US/_NF/store/uplink-catalog.ftl

```diff
@@ -151,6 +151,8 @@ uplink-security-emp-ammo-name = EMP Projectile Ammo
 uplink-security-emp-ammo-desc = A a box containing 4 EMP projectile grenades.
 uplink-security-trackingdart-ammo-name = Tracking Darts Ammo
 uplink-security-trackingdart-ammo-desc = A a box containing 4 tracking darts.
+uplink-security-spaceblade-sec-name = Security SpaceBlade
+uplink-security-spaceblade-sec-desc = Let it rip
```

Comment:
So is it SpaceBlade, Space Blade or space blade?
Also, a less blatant description :pray:

Reply:
_Let it slip_?  he he funny spaceman slip joke, doesn't break your descriptions.
_Spin it up_? _Gas and go_?

Reply:
fixed